### DISCUSSION TITLE
Define PR naming convention

### DISCUSSION THREAD

**Discussion by yhilmare** [oceanbase/odc#36] [General]
Let us define our pr naming convention, and there is a proposal:

```shell
[type][scope] Summary
```

Here is an example: https://github.com/apache/pulsar/pull/21048

* type: feat | improve | fix | cleanup | refactor | revert
* scope: connect | session | flow
* Summary: Explicit and self-explanatory. A complete sentence in present tense imperative. Capitalize 1st letter. No period at the end

**Comment by yhilmare** [comment]
Or we can maintain our current naming convention: 

```shell
type(scope): summary
```

**Comment by yhilmare** [comment]
There are two options:
1. [https://www.conventionalcommits.org/pt-br/v1.0.0/](https://www.conventionalcommits.org/pt-br/v1.0.0/)
2. [http://karma-runner.github.io/6.4/dev/git-commit-msg.html](http://karma-runner.github.io/6.4/dev/git-commit-msg.html)

**Comment by whhe** [comment]
I think we can keep the current naming convention since most GitHub actions on the marketplace work with it.
https://github.com/marketplace/actions/semantic-pull-request
https://github.com/marketplace/semantic-prs

**Reply by yhilmare** [reply]
Thanks 4 reply, we discussed offline and decided to keep our current pr template, just like this: http://karma-runner.github.io/6.4/dev/git-commit-msg.html
